pom.xml

```diff
@@ -76,6 +76,12 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.inject</groupId>
```

Comment:
This app fails to even compile. It uses `Types` here: https://github.com/cdap-guides/cdap-mapreduce-guide/blob/feature/CDAP-648-useWaitForStatus/src/main/java/co/cdap/guides/LogAnalyticsApp.java#L25 which comes from Guice.